TestProject/tests/output/PNGOutputTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Tests\Output;
+
+use GameOfLife\CellularAutomat\Board;
+use GameOfLife\Output\PNGOutput;
+use GameOfLife\Options\Getopt;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests whether \Output\PNGOutput works as expected.
+ */
+class PNGOutputTest extends TestCase
+{
+
+    protected $pngOutput;
+    protected $board;
+    protected $getOpt;
+
+    protected $cellColor = ["red" => 255, "green" => 255, "blue" => 255, "alpha" => 0];
+    protected $backgroundColor = ["red" => 0, "green" => 0, "blue" => 0, "alpha" => 0];
+
+    /**
+     * @return void
+     */
+    protected function setUp(): void
+    {
+        $this->board = new Board(5, 5);
+        $this->pngOutput = new PNGOutput();
+        $this->getOpt = $this->createMock(Getopt::class);
+
+        $this->currentDir = getcwd();
+        chdir("/tmp");
+        if (!is_dir("/tmp/imageOutput")) mkdir("/tmp/imageOutput");
+    }
+
+    /**
+     * @return void
+     */
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+
+        unlink("imageOutput/pngImage-000.png");
+        unlink("imageOutput/referenceImage.png");
+        rmdir("imageOutput/");
+        chdir($this->currentDir);
+    }
+
+    /**
+     * Asserts that the two png-images of the board aren't empty, by setting a background- and cell-color for each board.
+     *
+     * @test
+     */
+    public function boardsAreNotEmpty()
```

Comment:
Method name and documentation of class should be improved. What do you mean by an empty image? An Image with 0 Bytes size?